### PR TITLE
Ultrasonic sensor support

### PR DESCRIPTION
**Pull request by mikaukora**
Support for
- ultrasonic sensor parsing
- storing of unknown messages
- reprocessing of unknown messages

### CODE REVIEW

File: devices/parser/management/commands/parse_data.py [hunk 1]
```diff
@@ -30,69 +36,90 @@ def create_meta_field(timestamp, devid, devtype):
...
+        message["request"]["body"] = message["request"]["body"].decode()
+
+    return message
+
+
+def process_message(packed):
+    """ Process single packed raw message. """
+
+    data = data_unpack(packed)
+    print(data)
```
Comment (aapris) [inline devices/parser/management/commands/parse_data.py]: Or logging.debug() ?

File: persister/observations/models.py [hunk 2]
```diff
@@ -166,12 +166,10 @@ def save_data(message):
         time = None
         for items in data:
             if "time" in items:
-                # read the time entry for the measurements
-                print("Reading time for the measurements")
                 time = items["time"]
+                print(f"Measurement time: {time}")
```
Comment (aapris) [inline persister/observations/models.py]: Or logging.debug() ?

**Review by aapris — APPROVED** [review]
I commented a few print()s there, but it is your decision do you care. :)

**Comment by mikaukora** [comment]
> I commented a few print()s there, but it is your decision do you care. :)

Thanks, fixed.

**Pull request merged**